service/order.rest.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/rest-api-2.1.xsd"
+          name="tutorial" displayName="Order-Component REST API" version="2.0.0"
+          require-authentication="anonymous-all">
+    <resource name="orders">
+        <method type="post">
+            <service name="tutorial.order.OrderServices.create#OrderData"/>
+        </method>
+        <method type="post">
```

Comment:
We should not define multiple services/actions to be performed on a single method type. Here, for POST on /orders, the create#OrderData will never get called, as we defined new service call beneath it.